css-brochure-form/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div>
+      <h1>LEARN MORE</h1>
+    </div>
+    <div>
+      <form>
+        <div>
+          <label for="email-login"><strong>COMPANY EMAIL</strong></label>
```

Comment:
Using a `strong` element here is not ideal, since it will be read by screen readers as if it had major emphasis. You don't want your users that use screen readers to have "COMPANY EMAIL" screamed at them 😂 . Please revise your other labels as well. This is something that would be better done with styling in CSS.